### PR TITLE
Update logger to be consistent with serilog

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/logger.js [hunk 1]
```diff
@@ -1,64 +1,28 @@
...
+  format: combine(
+    label({ label: 'marketing-pages' }),
+    timestamp(),
+    colorize(),
+    logFormat,
+  ),
+  transports: [
+    new transports.Console({
+      level: loggerLevel,
+      colourize: (process.env.NODE_ENV === 'development' || !process.env.NODE_ENV),
```
Comment (alibombaye) [inline app/logger.js]: we could get this from the config?

Comment (kat1906) [reply app/logger.js]: yeah

Comment (alibombaye) [reply app/logger.js]: 😂 ok. Should we be using the config here then for env-vars rather than using them directly?

Comment (kat1906) [reply app/logger.js]: that's what I meant by yeah 😄

Comment (alibombaye) [reply app/logger.js]: I waited for it be resolved and it never came. So thought you just meant yeah as in yeah

**Pull request merged**